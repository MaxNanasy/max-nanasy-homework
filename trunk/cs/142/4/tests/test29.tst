type t1 = array 10 of short;
type t2 = array 10 of t1;
type t3 = array 10 of t2;

main()
{
  var a : t3;
  const c = 11;
  a[1][1] = c;
}
