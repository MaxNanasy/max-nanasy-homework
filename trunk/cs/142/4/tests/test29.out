Error: SEMANTIC: ARITY ERROR
***** Symbol Table Contents *****
N: int C: TYPE T: null
N: void C: TYPE T: null
N: short C: TYPE T: null
N: t1 C: TYPE T: short D: 10
N: t2 C: TYPE T: t1 D: 10
N: t3 C: TYPE T: t2 D: 10
N: MAIN C: PROCEDURE T: void
-->N: a C: VAR T: t3
-->N: c C: CONST T: short V: 11
